db/migrate/20200917140436_add_type_to_account.rb

```diff
@@ -0,0 +1,12 @@
+class AddTypeAndEncryptedDataToAccount < ActiveRecord::Migration[6.0]
+  def up
+    add_column :accounts, :type, :string, default: 'Account::Credentials', null: false
+    change_column :accounts, :type, :string, default: nil
```

Comment:
maybe we could just keep the default value? 
still thinking about anything that could go wrong if the type attr has a default value ... nothing coming up so far :joy_cat: